buildx.go

```diff
@@ -23,6 +24,15 @@ func cmdSetupBuildx(builder Builder, driverOpts []string) *exec.Cmd {
 	for _, opt := range driverOpts {
 		args = append(args, "--driver-opt", opt)
 	}
+	if harnessHttpProxy := os.Getenv("HARNESS_HTTP_PROXY"); harnessHttpProxy != "" {
+		args = append(args, "--driver-opt", fmt.Sprintf("env.http_proxy=%s", harnessHttpProxy))
+
+		if harnessHttpsProxy := os.Getenv("HARNESS_HTTPS_PROXY"); harnessHttpsProxy != "" {
+			args = append(args, "--driver-opt", fmt.Sprintf("env.https_proxy=%s", harnessHttpsProxy))
+		}
+
+		args = append(args, "--driver-opt", "network=host")
```

Comment:
Lets do this only if proxy is set. We can also expose it behind another env var since it might affect builds

Reply:
Can we test with and without this to make sure its definitely needed for Secure connect?